tests/scripts/run_demos.py

```diff
@@ -44,7 +44,7 @@ def run_all_demos(quiet=False):
 
     Return True if all demos passed and False if a demo fails.
     """
-    all_demos = glob.glob('programs/*/*_demo.sh')
+    all_demos = glob.glob('tf-psa-crypto/programs/*/*_demo.sh')
```

Comment:
With this when running `make neat; make -j; ./tests/scripts/run_demos.py`  programs/psa/key_ladder_demo.sh and programs/psa/psa_hash_demo.sh are run but not programs/test/dlopen_demo.sh. We should keep running the three.